dis_snek/models/discord/channel.py

```diff
@@ -1007,6 +1007,17 @@ def text_channels(self) -> List["TYPE_MESSAGEABLE_CHANNEL"]:
         """
         return [channel for channel in self.channels if isinstance(channel, GuildText)]
 
+    @property
+    def news_channels(self) -> List["GuildNews"]:
+        """
+        Get all text channels within the category.
```

Comment:
__news__

Reply:
copy-pasting is a skill that is clearly hard to master ;p